webui-page/webui.js

```diff
@@ -415,6 +426,38 @@ function setPosSlider(position, duration) {
   pos.innerHTML = format_time(slider.value);
 }
 
+function updatePosSlider(position) {
+  if(position === false){
+    return
+  }
+  const slider = document.getElementById("mediaPosition");
+  const pos = document.getElementById("position");
+  slider.value = position;
+  pos.innerHTML = format_time(slider.value);
+}
```

Comment:
It would be nice to use the `setPostSlider` function above for this. I've made some small changes to it, so you should better rebase first.